src/soilbiogeochem/SoilBiogeochemStateType.F90

```diff
@@ -132,26 +132,30 @@ subroutine InitHistory(this, bounds)
     begp = bounds%begp; endp= bounds%endp
     begc = bounds%begc; endc= bounds%endc
 
```

Comment:
Is this just a diagnostic used in BGC?  If so, how does FATES determine the vertical profile of these fluxes from multiple patches across a column?  I looks like these fluxes are vertically resolved in src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90, but I wanted to confirm.

Reply:
One of the compromises of FATES, is that it does not resolve soil dynamics at the patch scale, just litter. So when the litter fragments and become available to the decomposition scheme, it is all up-scaled to the single column element that the patches were on.  There is talk of expanding so that each fates site can work with multiple columns, so that we can capture variability in soil states.

Reply:
In the code none of these are used for FATES, so they would remain empty. They are initialized to spval, but for FATES we should probably set them to nan, so that they can't be used. I'll make an issue for that. The arrays still need to be allocated, because FATES does send them to fire interfaces -- but they are unused.

Reply:
#2063